client/components/ratings.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// this is just a stubout of the rating component
+
+const RatingStars = ({ stars }) => (
+  <>
+    {String.fromCharCode(9734) + String.fromCharCode(9734) + String.fromCharCode(9734) + String.fromCharCode(9734) + String.fromCharCode(9734)}
```

Comment:
Cool! String.fromCharCode(9734) is a star! How is that a star haha. Also when you have only one line of jsx I think the fragments are unnecessary.

Reply:
Good point!

Reply:
That's practically cheating! To think I spent all that time on SVG and polygons! Well, they still helped me with bars.